buttercup.el

```diff
@@ -1094,32 +1094,32 @@ responsibility to ensure ARG is a command."
                 nil))
             (_
              (error "Invalid `spy-on' keyword: `%S'" keyword)))))
-    (buttercup--spy-on-and-call-fake symbol replacement)))
+    (buttercup--spy-on-and-call-replacement symbol replacement)))
 
-(defun buttercup--spy-on-and-call-fake (spy fake-function)
-  "Replace the function in symbol SPY with a spy calling FAKE-FUNCTION."
+(defun buttercup--spy-on-and-call-replacement (spy fun)
```

Comment:
Wouldn't `(defun buttercup--spy-on (fun mock)` be more appropriate?

Reply:
The first arg is a symbol, not a function, and other functions consistently use `spy` for the argument name of the symbol whose function is being spied on, so I don't think it makes sense to change it here. As for the second arg name and the function name, I'm happy to call them whatever you think makes the most sense.

Reply:
Yeah, I noticed that it is called spy elsewhere and I find the naming confusing all around. But lets not bikeshed it, keep you change as it is.